packages/backend/src/utils/podman.ts

```diff
@@ -118,3 +132,48 @@ export async function isQEMUMachine(): Promise<boolean> {
 
   return false;
 }
+
+export async function checkContainerConnectionStatusAndResources(
+  memoryNeeded: number,
+): Promise<ContainerConnectionInfo> {
+  let connection: ProviderContainerConnection;
+  try {
+    connection = getFirstRunningPodmanConnection();
+  } catch (e) {
+    console.log(String(e));
+  }
+
+  if (!connection) {
+    return {
+      status: 'no-machine',
+    };
+  }
+
+  const engineInfo = await containerEngine.info(`${connection.providerId}.${connection.connection.name}`);
+  if (!engineInfo) {
+    return {
+      status: 'no-machine',
+    };
+  }
+
+  const hasCpus = engineInfo.cpus && engineInfo.cpus >= MIN_CPUS_VALUE;
```

Comment:
How do handle the case where the podman machine already has the maximum than the host, but still fell bellow the MIN you choice ?

Reply:
I think we can discuss about the minimum value but as we're not preventing the user from working even if its system does not have the minimum resources, we should be ok with it. If you have a machine with 6cpus and we suggest to have a machine with 10, you just keep doing your work ignoring it. If then, you see any performance issue, you know what could be the problem. Maybe we can add a setting to hide the warnings? 🤷‍♂️